### PR TITLE
Initial work on Popup docs

### PR DESCRIPTION
**Pull request by bijington**
## What changes to the docs does this PR provide?

Provides missing documentation for the `Popup` control.

## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] For new pages, used the [provided template](https://github.com/MicrosoftDocs/xamarin-communitytoolkit/blob/master/docs/.template.md).
- [x] For new features, added an entry in the [Table of Contents](https://github.com/MicrosoftDocs/xamarin-communitytoolkit/blob/master/docs/TOC.yml).
- [x] Ran against a spell and grammar checker.
- [x] Contains **NO** breaking changes.

### CODE REVIEW

**Comment by davidbritch** [comment]
Docs Build status updates of commit _[0650b58](https://github.com/bijington/xamarin-communitytoolkit/commits/0650b586a9f459318a6332f28b0252fc54ced2dc)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/TOC.yml](https://github.com/bijington/xamarin-communitytoolkit/blob/popup/docs/TOC.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/xamarin/community-toolkit/views/popup?branch=pr-en-us-141) |
[docs/views/popup-images/custom-positioning-popup.png](https://github.com/bijington/xamarin-communitytoolkit/blob/popup/docs/views/popup-images/custom-positioning-popup.png) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/xamarin/community-toolkit/views/popup?branch=pr-en-us-141) |
[docs/views/popup-images/simple-popup.png](https://github.com/bijington/xamarin-communitytoolkit/blob/popup/docs/views/popup-images/simple-popup.png) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/xamarin/community-toolkit/views/popup?branch=pr-en-us-141) |
[docs/views/popup.md](https://github.com/bijington/xamarin-communitytoolkit/blob/popup/docs/views/popup.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/xamarin/community-toolkit/views/popup?branch=pr-en-us-141) |

For more details, please refer to the [build report](https://opbuilduserstoragepublic.blob.core.windows.net/report/2021%5C11%5C23%5C52490272-7c0a-c2c6-3ec7-0fb104146cb5%5CPullRequest%5C202111232157344802-141%5Cworkflow_report.html?sv=2020-08-04&se=2021-12-24T21%3A58%3A18Z&sr=b&sp=r&sig=P5HyxWeyHBY29h804H2qXuKRLktRZwMUN%2BOFwpXixy8%3D).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/52490272-7c0a-c2c6-3ec7-0fb104146cb5?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching the docs.microsoft.com <a href="https://review.docs.microsoft.com/en-us/help/?branch=master">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

File: docs/views/popup.md [hunk 1]
```diff
@@ -0,0 +1,208 @@
...
+        <Label Text="Hello Simple Popup" />
+    </StackLayout>
+    
+</xct:Popup>
+```
+
+#### C#
+
+```csharp
+var popup = new Popup
```
Comment (pictos) [inline docs/views/popup.md:42]: I'm wondering if we should add the using namespace on the snippet
```suggestion
using Xamarin.CommunityToolkit.UI.Views;

var popup = new Popup
```

File: docs/views/popup.md [hunk 2]
```diff
@@ -0,0 +1,208 @@
...
+                Text = "Hello Simple Popup"
+            }
+        }
+    }
+};
+```
+
+### Showing the Popup
+
+With the Popup defined you will display it by using the `INavigation` extension methods. At the most basic level you can access this on any `ContentPage` in Xamarin.Forms via the following code:
```
Comment (pictos) [inline docs/views/popup.md]: I think is a good idea to mention that `ShowPopup` method is a extension method implemented on XCT

**Review by pictos — COMMENTED** [review]
@bijington this looks great, thanks for that! I just have some comments regarding the namespace

File: docs/views/popup.md [hunk 3]
```diff
@@ -0,0 +1,208 @@
+---
+title: Popup
+author: bijington
+ms.author: joverslu
+description: The Popup
+ms.date: 11/23/2021
+---
+
+# Title
```
Comment (jfversluis) [inline docs/views/popup.md]: I think something is missing here...

File: docs/views/popup.md (hunk 2 above)
Comment (jfversluis) [reply docs/views/popup.md]: Agree, let's mention explicitly that people need to add that namespace to avoid confusion

**Review by jfversluis — CHANGES_REQUESTED** [review]
Some minor things but I think this will be a great start to the popup docs!

File: docs/views/popup.md (hunk 3 above)
Comment (bijington) [reply docs/views/popup.md]: I can't tell you how long it took me to work out what was missing 🤣

**Comment by davidbritch** [comment]
Docs Build status updates of commit _[131df9e](https://github.com/bijington/xamarin-communitytoolkit/commits/131df9e524f8b670e4055ee1ebf052bf18d10558)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/TOC.yml](https://github.com/bijington/xamarin-communitytoolkit/blob/popup/docs/TOC.yml) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/xamarin/community-toolkit/views/popup?branch=pr-en-us-141) |
[docs/views/popup-images/custom-positioning-popup.png](https://github.com/bijington/xamarin-communitytoolkit/blob/popup/docs/views/popup-images/custom-positioning-popup.png) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/xamarin/community-toolkit/views/popup?branch=pr-en-us-141) |
[docs/views/popup-images/simple-popup.png](https://github.com/bijington/xamarin-communitytoolkit/blob/popup/docs/views/popup-images/simple-popup.png) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/xamarin/community-toolkit/views/popup?branch=pr-en-us-141) |
[docs/views/popup.md](https://github.com/bijington/xamarin-communitytoolkit/blob/popup/docs/views/popup.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/xamarin/community-toolkit/views/popup?branch=pr-en-us-141) |

For more details, please refer to the [build report](https://opbuilduserstoragepublic.blob.core.windows.net/report/2021%5C11%5C25%5C52490272-7c0a-c2c6-3ec7-0fb104146cb5%5CPullRequest%5C202111251406082883-141%5Cworkflow_report.html?sv=2020-08-04&se=2021-12-26T14%3A06%3A55Z&sr=b&sp=r&sig=c0WE7jQmrSnwAjLS0UGmcds4bEwNzGZ2WfOFGdLadnA%3D).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/52490272-7c0a-c2c6-3ec7-0fb104146cb5?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching the docs.microsoft.com <a href="https://review.docs.microsoft.com/en-us/help/?branch=master">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

**Review by jfversluis — APPROVED** [review]

**Pull request merged**